### PR TITLE
Fix possible null dereference

### PR DESCRIPTION
**Pull request by UselessToucan**

### CODE REVIEW

File: osu.Framework.Benchmarks/BenchmarkFontLoading.cs [hunk 1]
```diff
@@ -85,7 +85,7 @@ private void runFor(GlyphStore store)
                 {
                     var icon = (IconUsage)p.GetValue(null);
                     using (var upload = store.Get(icon.Icon.ToString()))
-                        Trace.Assert(upload.Data != null);
+                        Trace.Assert(upload != null && upload.Data != null);
```
Comment (bdach) [inline osu.Framework.Benchmarks/BenchmarkFontLoading.cs]: Might as well null propagate here?

**Comment by bdach** [comment]
Also note that adding nullchecks will trigger some null-reference tracking analyzer inspections that maybe don't really make sense (see CI).

File: osu.Framework/Platform/GameHost.cs [hunk 2]
```diff
@@ -798,7 +798,7 @@ protected virtual void SetupConfig(IDictionary<FrameworkSetting, object> gameDef
             bypassFrontToBackPass = DebugConfig.GetBindable<bool>(DebugSetting.BypassFrontToBackPass);
 
             threadLocale = Config.GetBindable<string>(FrameworkSetting.Locale);
-            threadLocale.BindValueChanged(locale =>
+            threadLocale?.BindValueChanged(locale =>
```
Comment (smoogipoo) [inline osu.Framework/Platform/GameHost.cs]: Config bindables should never be null. This should probably return an exception instead on the `GetBindable()` call.

File: osu.Framework/IO/Stores/RawCachingGlyphStore.cs [hunk 3]
```diff
@@ -48,7 +48,7 @@ private PageInfo createCachedPageInfo(int page)
 
             using (var stream = Store.GetStream(filename))
             {
-                string streamMd5 = stream.ComputeMD5Hash();
+                string streamMd5 = stream?.ComputeMD5Hash();
```
Comment (smoogipoo) [inline osu.Framework/IO/Stores/RawCachingGlyphStore.cs]: Should either fail completely or return null characters to match the general implementation of resource stores.

File: osu.Framework/Graphics/Textures/TextureAtlas.cs [hunk 4]
```diff
@@ -64,7 +64,7 @@ public void Reset()
             AtlasTexture = new TextureGLAtlas(atlasWidth, atlasHeight, manualMipmaps, filteringMode);
 
             using (var whiteTex = Add(WHITE_PIXEL_SIZE, WHITE_PIXEL_SIZE))
-                whiteTex.SetData(new TextureUpload(new Image<Rgba32>(SixLabors.ImageSharp.Configuration.Default, whiteTex.Width, whiteTex.Height, Rgba32.White)));
+                whiteTex?.SetData(new TextureUpload(new Image<Rgba32>(SixLabors.ImageSharp.Configuration.Default, whiteTex.Width, whiteTex.Height, Rgba32.White)));
```
Comment (smoogipoo) [inline osu.Framework/Graphics/Textures/TextureAtlas.cs]: Should fail completely, in the ctor.

File: osu.Framework/Graphics/Shaders/ShaderPart.cs [hunk 5]
```diff
@@ -114,7 +114,7 @@ private string loadFile(byte[] bytes, bool mainFile)
 
                         string backbufferCode = loadFile(manager.LoadRaw("sh_Backbuffer_Internal.h"), false);
 
-                        backbufferCode = backbufferCode.Replace("{{ real_main }}", realMainName);
+                        backbufferCode = backbufferCode?.Replace("{{ real_main }}", realMainName);
```
Comment (smoogipoo) [inline osu.Framework/Graphics/Shaders/ShaderPart.cs]: Should be asserted.

**Comment by smoogipoo** [comment]
Let's update these classes/methods to support NRT and revisit later.

**Pull request closed without merging**